docs/tutorials/resource-migration.md

```diff
@@ -0,0 +1,235 @@
+
+```shell
+$ kubectl --context member1 apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+Thus, we can use `member1` as the cluster with existing resources, while `member2` as a bare cluster.
+
+## Tutorials
+
+### Migrate all the resources to Karmada
+
+1）Apply the local file `/tmp/deployments-and-services.yaml` written just now to Karmada control plane too.
+
+**Step 5: Run the command**
+
+```shell
+➜ ✗ kubectl --context karmada-apiserver apply -f /tmp/deployments-and-services.yaml
+deployment.apps/nginx-deploy created
+service/nginx-svc created
+deployment.apps/hello-deploy created
+service/hello-svc created
+```
+
+2）Write a `PropagationPolicy` to the local file `/tmp/pp-for-migrating-deployments-and-services.yaml`.
+
+**Step 6: Write the code**
+
+Create new file `/tmp/pp-for-migrating-deployments-and-services.yaml` and copy text below to it:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: migrate-pp
+spec:
+  conflictResolution: Overwrite
+  placement:
+    clusterAffinity:
+      clusterNames:
+      - member1
+  priority: 0
+  resourceSelectors:
+  - apiVersion: apps/v1
+    kind: Deployment
+  - apiVersion: v1
+    kind: Service
+  schedulerName: default-scheduler
+```
+
+**Note:**
+
+You should pay attention to two fields： 
+
+* `spec.conflictResolution: Overwrite`：the value must be [Overwrite](https://github.com/karmada-io/karmada/blob/master/docs/proposals/migration/design-of-seamless-cluster-migration-scheme.md#proposal).
+* `spec.resourceSelectors`：selecting resources to migrate, you can define your custom [ResourceSelector](https://karmada.io/docs/userguide/scheduling/override-policy/#resource-selector).
+
+3）Apply `/tmp/pp-for-migrating-deployments-and-services.yaml` to Karmada control plane. 
+
+**Step 7: Run the command**
+
+```shell
+$ kubectl --context karmada-apiserver apply -f /tmp/pp-for-migrating-deployments-and-services.yaml
+propagationpolicy.policy.karmada.io/migrate-pp created
+```
+
+Now, you have finished the migration, isn't it so easy? 
+
+4）Verification
+
+**Step 8: Run the command**
+
+```shell
+$ kubectl --context karmada-apiserver get deploy
+$ kubectl --context karmada-apiserver get rb
+```
+
+You can see the Deployments in Karmada are all ready and the `aggregatedStatus` of `ResourceBinding` is applied.
+
+### Apply higher priority PropagationPolicy
+
+1）Write a high priority `PropagationPolicy` to the local file `/tmp/pp-for-nginx-app.yaml`.
+
+**Step 9: Write the code**
+
+Create new file `/tmp/pp-for-nginx-app.yaml` and copy text below to it:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-pp
+spec:
+  conflictResolution: Overwrite
+  placement:
+    clusterAffinity:
+      clusterNames:
+      - member1
+      - member2                          ## focus on this line
```

Comment:
`## focus on this line` this comment seems for `priority: 10`.

Reply:
No, I meant the line `- member2` is newly added, I will update the comment more clear.